modules/cassandrabackend/test/pdns.conf

```diff
@@ -0,0 +1,552 @@
+# Autogenerated configuration file template
```

Comment:
You can remove the commented out lines from your config file, they are not necessary for testing.